### ISSUE TITLE
Nvda does not read the name of the edit field.

### ISSUE THREAD

**Issue by fernando-jose-silva** [nvaccess/nvda#6817]
He was doing tests on the accessibility of a Portuguese language site. Brazil:
Http://3remetodista.org.br/
And identified a problem that yoke can be solved by nvda.
Using nvda next 13840 and firefox 51.01 using chrome 56 the result is the same.
When you open the site and walk with arrow below just after the text:
Nunca perca uma atualização 

And continue navigating with the down arrow in the first edit field, nvda does not read the field label.
Nvda reads only "edit field".
The expected would be that the nvda reads "edit field name".
When pressing another arrow below is another field of editing, that nvda reads only the edit field, and the expected would be that it reads:
Email editing field.
If you press nvda + tab, you can read these labels from the edit fields.

**Comment by ehollig** [comment]
@fernando-jose-silva, is this still happening for you? When I go to the URL mentioned and go to the edit field, NVDA reads:
Subscribe form landmark edit  has auto complete
Subscribe  form
Nome  edit  has auto complete
I tested this in Firefox, IE, and Chrome with similar results.

**Comment by bhavyashah** [comment]
Confirmed. Using Firefox 54 stable and NVDA 2017.2 on a Windows 8.1 system, I can successfully replicate the reported issue with the following STR:
* Go to 
* Press NVDA+Ctrl+F, copy and paste 'Nunca perca uma atualização' as the text you wish to find, and hit Enter to land on the required instance of the entered string.
* Assuming screen layout is off, press down arrow once.
Expected Result: NVDA should report the edit field label as it does when the edit field is Tabbed to as 'Name' followed by 'edit has auto complete'.
Actual Result: NVDA reports 'edit has auto complete' only, leaving the user to wonder the purpose of the edit field.
Something very similar happens with the e-mail address edit field just below this one also. Regardless, I am fairly positive that @jcsteh has interacted with another ticket concerning this same bug, which means that this ticket may be a duplicate of another. If not, I would strongly recommend including this as part of Project Webfix, because this is a frequently encountered problem when filling a web form, something that is common and is faced in the wild.

**Comment by jcsteh** [comment]
Duplicate of #4773.
